v22.2/install-cockroachdb-mac.md

```diff
@@ -21,8 +21,17 @@ docs_area: deploy
 
 Use one of the options below to install CockroachDB.
 
+{% capture arch_note_homebrew %}For CockroachDB v22.x and above, Homebrew installs binaries for your system architecture, either Intel or ARM ([Apple Silicon](https://support.apple.com/en-us/HT211814)). ARM binaries are **experimental** and are not qualified for production use.
```

Comment:
> I don't think we're using experimental anymore, although I'm not sure how much the new preview nomenclature fits.

Ah! We in dev inf were not aware that there was a name change afoot here. We had understood that "experimental" was applied to the Windows cockroach binaries. That word also sounds appropriate for the state of the ARM64 Linux binaries for the v22.2 release series since we will not be supporting them for production use.

For the Mac binaries, we didn't used to use "experimental" for Intel, but they are not supported for production use. Our (the dev inf team's) thought here is that the ARM64 binaries should follow a similar pattern: they don't get marked experimental and they are not supported for production use.

Note that we have already publicly been using the "experimental" word to describe our support level for the ARM64 Linux binaries. See these issue comments: [one](https://github.com/cockroachdb/cockroach/issues/60778#issuecomment-1201704432), [two](https://github.com/cockroachdb/cockroach/issues/32689#issuecomment-1260298632). It may also have been used with some existing clients.

All that said, if there is an alternative word that we should be using to describe these builds, then we can probably still change it, make a few targeted updates to publicly accessible places it's already noted as experimental and communicate that change in wording internally.

Reply:
I synced with @taroface offline. "Experimental" is the correct term to use for binaries that aren't yet qualified for production.

Currently, all Windows binaries are marked as "experimental", as well as Linux ARM binaries. Given that, is this a request to label all Mac binaries as "experimental" or to remove it from the Linux ARM binaries (and presumably Windows) or to commit to some ambiguity about Linux ARM vs all the others?

Reply:
I fixed the paragraph-rendering issue by converting the Liquid variables to direct HTML instead of Markdown, although @nickvigilante and I aren't quite sure why.